tests/system/test_diags.py

```diff
@@ -169,15 +169,22 @@ def check_html_image(self, html_path, png_path, full_png_path):
         path_to_actual_png = full_png_path
         path_to_expected_png = "../unit_test_images/{}".format(png_path)
 
-        mismatched_images = []  # type: ignore
-        compare_images(
-            self,
-            mismatched_images,
-            image_name,
-            path_to_actual_png,
-            path_to_expected_png,
-        )
-        self.assertEqual(mismatched_images, [])
+        try:
+            # Set `export CHECK_IMAGES=True` to do a pixel-by-pixel image comparison check.
+            check_images = bool(os.environ["CHECK_IMAGES"])
+        except KeyError:
+            check_images = False
```

Comment:
@chengzhuzhang I set the default value as `False` because I believe we then wouldn't have to update the feedstock to set the value to `False`. Is that correct, @xylar? However, setting it to `True` would make it easier for testers because the image checks would automatically be run.

Once we decide on the default value I can 1) squash the commits and merge, 2) make a version update commit, 3) release `rc2`.

Reply:
I haven't tested the change, but the PR looks fine. Just need to remember not to include `export CHECK_IMAGES=True` in CI for feedstock.

Reply:
I agree that that `True` is the better default.  We can set `CHECK_IMAGES=False` in the feedstock.